slice/Ice/Ice2Definitions.ice

```diff
@@ -40,6 +40,7 @@ module Ice
         string operation;
         bool? \idempotent;       // null equivalent to false
         Priority? priority;      // null equivalent to 0
+        varlong deadline;
```

Comment:
varlong suggests we use negative values - typically to use "-1" to mean infinite. If we don't, it should be varulong.

Reply:
keep it as varlong, as we will use -1 for infinite deadline